src/build.h

```diff
@@ -227,6 +226,13 @@ struct Builder {
                     const string& deps_prefix, vector<Node*>* deps_nodes,
                     string* err);
 
+  /// Map of running edge to time the edge started running.
+  typedef map<Edge*, int> RunningEdgeMap;
```

Comment:
Similar to my comment about "EdgePtrSet", I would personally think it an improvement to name this type "RunningEdgePtrMap", or "RunningEdgePtrMap_t"

Reply:
RunningEdgePtrMap_t is out due to the google naming convention.
I don't care much between RunningEdgeMap and RunningEdgePtrMap. If anyone else has strong feelings on this please chime in, otherwise I'll leave it as-is to avoid extra work.

Reply:
Given that this isn't new code, but just moved, I would say renaming this doesn't need to be part of this commit. The typedef will be less needed anyway, when we can use `auto` in the future.